.github/workflows/cmake.yml

```
@@ -954,10 +957,18 @@ jobs:
-        COVERALLS_REPO_TOKEN: "${{ secrets.COVERALLS_REPO_TOKEN }}"
+  finish:
+    needs: [cmake]
+    runs-on: ubuntu-slim
+    if: |
+      always()
+      && !contains(needs.cmake.result, 'failure')
+      && !contains(needs.cmake.result, 'cancelled')
+    steps:
+      - name: Upload to Coveralls - Final
+        uses: coverallsapp/github-action@v2
+        if: (secrets.COVERALLS_REPO_TOKEN != '' || github.repository == 'zlib-ng/zlib-ng')
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)